src/constructs/loadbalancing/clb.ts

```diff
@@ -1,33 +1,50 @@
-import type { CfnLoadBalancer, LoadBalancerProps } from "@aws-cdk/aws-elasticloadbalancing";
-import { LoadBalancer } from "@aws-cdk/aws-elasticloadbalancing";
+import type { CfnLoadBalancer, HealthCheck, LoadBalancerProps } from "@aws-cdk/aws-elasticloadbalancing";
+import { LoadBalancer, LoadBalancingProtocol } from "@aws-cdk/aws-elasticloadbalancing";
+import { Duration } from "@aws-cdk/core";
 import type { GuStack } from "../core";
 
 enum RemoveableProperties {
   SCHEME = "Scheme",
 }
 
-interface GuClassicLoadBalancerProps extends LoadBalancerProps {
+interface GuClassicLoadBalancerProps extends Omit<LoadBalancerProps, "healthCheck"> {
   overrideId?: boolean;
   propertiesToRemove?: RemoveableProperties[];
   propertiesToOverride?: Record<string, unknown>;
+  healthCheck?: Partial<HealthCheck>;
 }
 
 export class GuClassicLoadBalancer extends LoadBalancer {
   static RemoveableProperties = RemoveableProperties;
 
+  static DefaultHealthCheck = {
+    port: 9000,
+    path: "/healthcheck",
+    protocol: LoadBalancingProtocol.HTTP,
+    healthyThreshold: 2,
+    unhealthyThreshold: 5,
+    interval: Duration.seconds(30),
+    timeout: Duration.seconds(10),
+  };
```

Comment:
These seemed like sensible defaults at first pass but keen to hear any opinions on what these values should be.